### PR TITLE
Osra410 adminuser delete

### PR DESCRIPTION
**Pull request by robbiejackson**

### CODE REVIEW

File: spec/views/hq/admin_users/new.html.erb_spec.rb [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    render
+    expect(view).to render_template(:new)
+    expect(rendered).to have_selector("form")
+    ["email", "password", "password_confirmation"].each do |field|
+      expect(rendered).to have_selector("input[id='admin_user_#{field}']")
+    end
+    expect(rendered).to have_selector("input[type='submit'][value='Create Admin User']")
+  end
+
+  it 'has a cancel button going to the index page' do
```
Comment (gmunumel) [inline spec/views/hq/admin_users/new.html.erb_spec.rb]: Why are we testing only a cancel button?

Comment (robbiejackson) [inline spec/views/hq/admin_users/new.html.erb_spec.rb]: That's the only other button there is. 
The submit button is tested in the previous test, where it checks that all the standard form elements (form tag, fields and submit button) are present.

**Comment by razorcd** [comment]
please rebase this one too

**Comment by robbiejackson** [comment]
OSRA 415 AdminUser delete now rebased.

**Pull request merged**